src/context/index.ts

```diff
@@ -128,3 +150,10 @@ export const setupContext = async (config: Config): Promise<DirectoryContext | Y
     `Unable to determine context processor to load for file ${inputFile}, does it exist? `
   );
 };
+
+export const filterOnlyIncludedResourceTypes =
```

Comment:
Debatably overkill to pull into its own function, but this filtering function was being used in several foundational areas of code. I think that encapsulating the logic of it is important. Also, it enables additional unit testing.